File: 2021.2.5/1.c
// Задача 1.Напишете функцията void printStr(char *s) аргумент поинтер
// към началото на стринга. Използвайте for цикъл и обходете стринга докато
// не стигнете знака за на стринг '\0'. На ваяка итерация пристирайте всеки
// един елемент на стринга с printf(“%с”, с).

#include <stdio.h>
void printArr(char a[]);

int main(){
 char str[] = "Hello world";
 printArr(str);
 return 0;
}

 void printArr(char* a){

    int i = 0;
    for (int i = 0; a[i]!='\0'; i++)
    {
       printf("%c",a[i]);
    }
    
     
 }

File: Python/25.4.2021/1.py
from functools import reduce


def my_sum(*args):
    print(reduce((lambda x, y: x + y), args))


my_sum("Hello ", "World")
my_sum("my ", "name ", "is ", "martin")
my_sum(2, 3, 2)
my_sum(1, 2, 3, 5, 6, 5, 9, 7, 9, 100, 1, 3, 2, )
my_sum(2.2, 3.4, 2.3, 5, 6, 8, )
